.gitignore

```diff
@@ -1,21 +1,104 @@
-*.egg-info
-*.pyc
-*.pyo
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
 *$py.class
+
+# C extensions
 *.so
-__pycache__
-.coverage
-.installed.cfg
-.mr.developer.cfg
```

Comment:
i think this one should be copied over

Reply:
add it to the botom with a comment `# mr.developer` (it's apparently part of this: https://pypi.python.org/pypi/mr.developer